test/test_poolmanager.py

```diff
@@ -474,3 +475,18 @@ def test_thread_safty(self) -> None:
 
         # Connection should be closed, because reference to pool_1 is gone.
         assert conn_queue.qsize() == 0
+
+    @pytest.mark.skipif(not hasattr(sys, "audit"), reason="Requires python 3.8+")
+    def test_http_client_connect_audit_event(self) -> None:
+        event_name = None
+
+        def _hook(event: str, args: tuple) -> None:  # type: ignore[type-arg]
+            if event == "http.client.connect":
+                nonlocal event_name
+                if event == "http.client.connect":
```

Comment:
Why check this again?

Reply:
Ohh, I missed it, Thanks for review @sigmavirus24